### DISCUSSION TITLE
directus bootstrap / directus install ,custom migrations failed

### DISCUSSION THREAD

**Discussion by korneliusz-w** [directus/directus#7342] [Archive]
Hi

I try to run npx directus bootstrap or npx directus install. 
After installing Directus system tables its run custom migrations but it failed. Column "accountability" is correct but it is not exist in this moment in "directus_collections" table, it will be in the future of installation process.

How to solve this problem?

npx directus bootstrap
09:18:01 ÔťĘ Initializing bootstrap...
09:18:01 ÔťĘ Installing Directus system tables...
09:18:01 ÔťĘ Running migrations...
✨ Applying Create Table Business Page...
✨ Applying Table Business Page Collection Definition...
error: insert into "directus_collections" ("accountability", "archive_app_filter", "archive_field", "archive_value", "collection", "color", "display_template", "hidden", "icon", "item_duplication_fields", "note", "singleton", "sort_field", "translations", "unarchive_value") values ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14, $15) - column "accountability" of relation "directus_collections" does not exist

**Comment by rijkvanzanten** [comment]
@korneliusz-w I think your custom migrations are ran before the system migrations, meaning that that column doesn't exist yet at the time your migrations are run. Try using a different ID in your custom migration file (the first bit of the file name `20210811A-abc-def.js`
